inference-engine/tests/ie_test_utils/functional_test_utils/CMakeLists.txt

```diff
@@ -4,24 +4,32 @@
 
 set(TARGET_NAME funcTestUtils)
 
-list(APPEND EXPORT_DEPENDENCIES inference_engine_lp_transformations
-    commonTestUtils inference_engine inference_engine_legacy)
+list(APPEND EXPORT_DEPENDENCIES
+        inference_engine_lp_transformations
+        commonTestUtils
+        inference_engine
+        inference_engine_legacy
+        )
 
 addIeTarget(
     NAME ${TARGET_NAME}
     TYPE STATIC
     ROOT ${CMAKE_CURRENT_SOURCE_DIR}
     ADD_CPPLINT
     DEVELOPER_PACKAGE
-    LINK_LIBRARIES ngraphFunctions
-    EXPORT_DEPENDENCIES ${EXPORT_DEPENDENCIES}
+    LINK_LIBRARIES
+        PUBLIC
+            ${EXPORT_DEPENDENCIES}
+            ngraphFunctions
```

Comment:
The fix is not correct. You still add nGraphFunctions as a public dependency.

Reply:
I will review it in the next PR.